src/components/TopMenu.svelte

```diff
@@ -138,12 +163,21 @@
       uk-tooltip
       disabled={$activeDatasets.length < 2}><Fa icon={faChartLine} /></button
     >
+  </div>
+  <div class="uk-button-group" data-tour="scale">
+    <button
+      type="button"
+      class="uk-button uk-button-default uk-button-small"
+      on:click|preventDefault={scaleMean}
+      title="Scale by 1/mean"
+      uk-tooltip><Fa icon={faMinimize} /></button
```

Comment:
```suggestion
      uk-tooltip><Fa icon={faAnchor} /></button
```
Can we keep the old anchor icon for this, and use this new icon for the "autofit mode" button?